workers/email/src/templates/emails/assignee/text.twig

```diff
@@ -10,8 +10,8 @@ View event: {{ host }}/project/{{ project._id }}/event/{{ event._id }}/overview
 
 ***
 
-You received this email because you are currently opted in to receive such alerts via your project’s notifications settings. You may adjust your preferences at any time by clicking the link above. To unsubscribe, follow the link:
-{{ host }}/unsubscribe/{{ project._id }}
+You received this email because you are currently opted in to receive such alerts via your personal notifications settings. You may adjust your preferences at any time by clicking the link above. To unsubscribe, follow the link:
```

Comment:
```suggestion
You received this email because you are currently opted in to receive such alerts via your personal notifications settings. You may adjust your preferences at any time by clicking the link:

```